app/javascript/redux/greetingsSlice.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const url = 'http://localhost:5000/api/greetings';
+
+const initialState = {
+  greetings: [],
+  isLoading: true,
+};
+
+export const getGreetings = createAsyncThunk(
+  'greetings/getGreetings',
+  async () => {
+    const response = await fetch(url);
+    const data = await response.json();
+    console.log(data);
```

Comment:
- [ ] @Rachelwebdev great work on your greetings reducer, however, it's not good practice to have a console.log statement in your codebase kindly remove it, it's a security threat as unauthorized users may gain access to sensitive data through it displaying data on the browser.
- [ ] @Rachelwebdev Your app is functioning well without errors but could not get it to display greetings, as shown below I can get a random greeting using API but in the app root route it's not displaying the greetings, kindly consider fixing it
![image](https://github.com/Rachelwebdev/greeting-rails-react/assets/61408860/69bb359d-d855-49fd-8877-87cfa9323673)
![image](https://github.com/Rachelwebdev/greeting-rails-react/assets/61408860/90069da3-13ee-49dc-be0b-7a1cf2977902)